packages/plugin-lightningcss/src/index.ts

```diff
@@ -0,0 +1,7 @@
+export { pluginLightningcss } from './plugin';
+export { default as lightningcssLoader } from './loader';
+export { LightningCssMinifyPlugin } from './minimizer';
```

Comment:
when will users use the `lightningcssLoader` and `LightningCssMinifyPlugin`, can they be priviate?